var/spack/repos/builtin/packages/cgdcbxd/package.py

```diff
@@ -17,13 +17,21 @@ class Cgdcbxd(AutotoolsPackage):
     version('1.0.2', sha256='ef626c60e27005d3cae1e19a60d0133be0d1f0a012b695f7f1f6ad5a2afa4166')
     version('1.0.1', sha256='663b87a5ab4a760e2446e479fec36f6300d53e6113af1408efc156325219900c')
 
+    variant('system_install', default=False, description=(
+        'install config into system'))
```

Comment:
I think we should have no variant and always assume `~system_install`. How can we properly uninstall something installed in system folders?

Reply:
Yes, I also think that we should never touch the host system.

Pushed an update accordingly to remove the variant and use `DESTDIR=self.prefix` for installation always!